src/adapters/scene.js

```diff
@@ -431,4 +452,8 @@ Scene.prototype.mobileButtonVisibility = function(selector, visible) {
   }
 };
 
+Scene.prototype.cancelLongTouch = () => {
```

Comment:
See the comment above. You don't act on any part of the object here so it doesn't need to be a class method.